CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Vis Builder] Add field summary popovers ([#2682](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2682))
 - [I18n] Register ru, ru-RU locale ([#2817](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2817))
 - Add yarn opensearch arg to setup plugin dependencies ([#2544](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2544))
-
+- [Multi DataSource] Add Test connection feature on data source create and update ([#2973](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2973))
```

Comment:
This needs rewording. Maybe `Test the connection to an external data source when creating or updating`.

Also, please add a blank line after it.